README.md

```diff
@@ -139,6 +139,10 @@ Enable headers for request limit (`X-RateLimit-Limit`) and current usage (`X-Rat
 
 Defaults to `true`.
 
+### draft_polli_ratelimit_headers
+
+This plugin is to support (eg. eventually via configuration) the new (ratelimit standardization proposal)[https://tools.ietf.org/id/draft-polli-ratelimit-headers-01.html].
```

Comment:
Lets reword this to something like

> Enable headers conforming to the (ratelimit standardization proposal)[https://tools.ietf.org/id/draft-polli-ratelimit-headers-01.html]: 'RateLimit-Limit`, `RateLimit-Remaining`, and, if the store supports it, `RateLimit-Reset`. May be used in conjunction with, or instead of the `headers` option.

That also prompts the question: what should we do if the store doesn't tell us when the rate limit will reset? I'll bring that up in the original ticket.

Reply:
The spec [does not  mandate](https://ioggstream.github.io/draft-polli-ratelimit-headers/draft-polli-ratelimit-headers.html#providing-ratelimit-headers) the usage of all headers.

```
A server MAY use one or more RateLimit response header fields defined in this document to communicate its quota policies.
```

>  what should we do if the store doesn't tell us when the rate limit will reset? 

[here](https://ioggstream.github.io/draft-polli-ratelimit-headers/draft-polli-ratelimit-headers.html#ratelimit-limit-header)

```
If the expiring-limit is not associated to a time-window, the time-window MUST either be:

- inferred by the value of RateLimit-Reset at the moment of the reset, or
- communicated out-of-band (eg. in the documentation). <--- this is the Memcache
```

Reply:
Cool, I put a note on the original ticket.

Also, @sanyamdogra I tweaked the proposed text in my earlier comment a bit, and made it copy-past-able.

Reply:
Alright sure!